luigi/db_task_history.py

```diff
@@ -187,6 +187,13 @@ def find_task_by_id(self, id, session=None):
         with self._session(session) as session:
             return session.query(TaskRecord).get(id)
 
+    def find_task_by_task_id(self, task_id, session=None):
+        """
+        Find task with the given task ID.
+        """
+        with self._session(session) as session:
+            return session.query(TaskRecord).filter(TaskRecord.task_id==task_id).all()[-1]
```

Comment:
flake8 tests complain about this line.

```
./luigi/db_task_history.py:195:71: E225 missing whitespace around operator
```

Reply:
@dlstadther , could you tell us what needs to be corrected here? Should we put space around `==`?

Reply:
@dlstadther done. Anything that needs to be added?

@tashrifbillah that should be it.

Reply:
Thanks @RRap0so !